NEWS

```diff
@@ -177,6 +177,15 @@ v2.0.0 final release; see:
   Daryin for reporting this issue.
 - Fix various a compilation problems on Cygwin.  Thanks to Marco Atzeri
   for supplying these fixes.
+- Fix for opal pkg-configure file.  Thanks to Mark Santcroos for reporting.
+- Fix a problem with MPI_Win_allocate.  Thanks to Peter Wind for reporting.
```

Comment:
Probably should be in all caps to be language neutral (MPI_WIN_ALLOCATE), like you did with MPI_COMBINER_RESIZED, below.

Also, in prior NEWS groups, we tended to put the items in (more or less) reverse-chronological order.  I don't know if that really matters, but that's the way it was (generally) done before.